homeassistant/components/private_ble_device/sensor.py

```diff
@@ -73,6 +76,21 @@ class PrivateDeviceSensorEntityDescription(
         device_class=SensorDeviceClass.DISTANCE,
         suggested_display_precision=1,
     ),
+    PrivateDeviceSensorEntityDescription(
+        key="estimated_broadcast_interval",
+        translation_key="estimated_broadcast_interval",
+        icon="mdi:timer-sync-outline",
+        native_unit_of_measurement=UnitOfTime.SECONDS,
```

Comment:
This should probably be marked as `EntityCategory.DIAGNOSTIC`

Reply:
Grr good spot. It was certainly meant to be.